superset-frontend/src/components/Select/utils.tsx

```diff
@@ -121,104 +65,7 @@ export function hasOption(
   );
 }
 
-export type AntdProps = AntdSelectProps<AntdSelectValue>;
-
-export type AntdExposedProps = Pick<
-  AntdProps,
-  | 'allowClear'
-  | 'autoFocus'
-  | 'disabled'
-  | 'filterOption'
-  | 'filterSort'
-  | 'loading'
-  | 'labelInValue'
-  | 'maxTagCount'
-  | 'notFoundContent'
-  | 'onChange'
-  | 'onClear'
-  | 'onDeselect'
-  | 'onSelect'
-  | 'onFocus'
-  | 'onBlur'
-  | 'onPopupScroll'
-  | 'onSearch'
-  | 'onDropdownVisibleChange'
-  | 'placeholder'
-  | 'showArrow'
-  | 'showSearch'
-  | 'tokenSeparators'
-  | 'value'
-  | 'getPopupContainer'
-  | 'menuItemSelectedIcon'
->;
-
-export type SelectOptionsType = Exclude<AntdProps['options'], undefined>;
-
-export type SelectOptionsTypePage = {
-  data: SelectOptionsType;
-  totalCount: number;
-};
-
-export type SelectOptionsPagePromise = (
-  search: string,
-  page: number,
-  pageSize: number,
-) => Promise<SelectOptionsTypePage>;
-
-export const StyledContainer = styled.div`
-  display: flex;
-  flex-direction: column;
-  width: 100%;
-`;
-
-export const StyledSelect = styled(AntdSelect)`
-  ${({ theme }) => `
-    && .ant-select-selector {
-      border-radius: ${theme.gridUnit}px;
-    }
-    // Open the dropdown when clicking on the suffix
-    // This is fixed in version 4.16
-    .ant-select-arrow .anticon:not(.ant-select-suffix) {
-      pointer-events: none;
-    }
-  `}
-`;
-
-export const StyledStopOutlined = styled(Icons.StopOutlined)`
-  vertical-align: 0;
-`;
-
-export const StyledCheckOutlined = styled(Icons.CheckOutlined)`
-  vertical-align: 0;
-`;
-
-export const StyledSpin = styled(Spin)`
-  margin-top: ${({ theme }) => -theme.gridUnit}px;
-`;
-
-export const StyledLoadingText = styled.div`
-  ${({ theme }) => `
-    margin-left: ${theme.gridUnit * 3}px;
-    line-height: ${theme.gridUnit * 8}px;
-    color: ${theme.colors.grayscale.light1};
-  `}
-`;
-
-const StyledHelperText = styled.div`
-  ${({ theme }) => `
-    padding: ${theme.gridUnit * 2}px ${theme.gridUnit * 3}px;
-    color: ${theme.colors.grayscale.base};
-    font-size: ${theme.typography.sizes.s}px;
-    cursor: default;
-    border-bottom: 1px solid ${theme.colors.grayscale.light2};
-  `}
-`;
-
-export const MAX_TAG_COUNT = 4;
-export const TOKEN_SEPARATORS = [',', '\n', '\t', ';'];
-export const EMPTY_OPTIONS: SelectOptionsType = [];
-
-export const DEFAULT_SORT_COMPARATOR = (
+export const defaultSortComparator = (
```

Comment:
Why the change? I thought we want all caps for constants?

Reply:
Kind of feel this needs to be in `constants.ts`, too.

Reply:
I was in doubt if I should treat it as a constant or another utility function. I'll revert the change and add it to the constants file.